ui/src/app/webpack.config.js

```diff
@@ -24,7 +24,7 @@ const config = {
     path: __dirname + "/../../dist/app"
   },
 
-  devtool: "source-map",
+  devtool: isProd ? false : "source-map",
```

Comment:
```suggestion
  devtool: isProd ? false : "eval",
```

If we only have dev source maps, there are much faster options. `eval` is [recommended](https://webpack.js.org/configuration/devtool/) for dev

Reply:
Great. I also think eval is better for development